src/io/jesi/backpack/test/strict.cljc

```diff
@@ -0,0 +1,106 @@
+;TODO move to test utils library
+(ns io.jesi.backpack.test.strict
+  (:refer-clojure :exclude [=])
+  #?(:cljs (:require-macros [io.jesi.backpack.test.strict :refer [is]]))
+  (:require
+    [clojure.string :as str]
+    [clojure.test :as test]
+    [io.jesi.backpack.env :as env])
+  #?(:clj (:import
+            (java.util.regex Pattern))))
+
+(defn =
+  #?(:clj
+     "Equality. Takes at least 2 args. Returns `true` if `x` equals `y`, `false` if not.
+     Same as Java `x.equals(y)` except it also works for `nil`, and compares numbers
+     and collections in a type-independent manner. Clojure's immutable data structures
+     define `equals()` (and thus `=`) as a value, not an identity, comparison."
+     :default
+     "Equality. Takes at least 2 args. Returns `true` if `x` equals `y`, `false` if not.
+     Compares numbers and collections in a type-independent manner. Clojure's immutable
+     data structures define `-equiv` (and thus `=`) as a value, not an identity, comparison.")
+  [x y & more]
+  (if (seq more)
+    (apply clojure.core/= x y more)
+    (clojure.core/= x y)))
+
+(defmacro is
+  "Generic assertion macro. `form` is any predicate test.
+  `msg` is an optional message to attach to the assertion.
+  Will fail is message is not a string.
+
+  Example: `(is (= 4 (+ 2 2)) \"Two plus two should be 4\")`
+
+  Special forms:
+
+  `thrown?` checks that an instance of `c` is thrown from
+  body, fails if not; then returns the thing thrown.
+
+  `thrown-with-msg?` checks that an instance of `c` is
+  thrown from `body` and that the message on the exception
+  matches (with `re-find`) the regular expression `re`."
+  ([form]
+   {:pre [(some? form)]}
+   `(env/transform
+      (test/is ~form)))
+  ([form msg]
+   {:pre [(some? form)]}
```

Comment:
This was broken before...
A macro `:pre` receives quotes symbols, never a string